timemachine/cpp/src/nonbonded_all_pairs.hpp

```diff
@@ -19,6 +19,8 @@ template <typename RealType> class NonbondedAllPairs : public Potential {
     double beta_;
     double cutoff_;
 
+    unsigned int steps_;
```

Comment:
Please rename this to `steps_since_last_sort_` or something, we have overload the tern `steps` way too much

Reply:
(or `num_times_called_`?)

Reply:
Changed name and added comment in https://github.com/proteneer/timemachine/pull/1042/commits/dca5d8f834d478fd269758072bfd1a2d349bf685